### PR TITLE
Requesting to Merge: Improve UI to Birthday Website

### LINKED ISSUE #1013: Improve UI to Birthday Website
**Describe the Feature you want to end**
This issue is a request to enhance the user interface (UI) of the website

**Expected behavior**
The current UI could benefit from improvements to enhance the overall user experience and make the website more visually appealing.

**Screenshots (optional)**
birthday

**Additional context (optional )**

### PR DESCRIPTION
**Pull request by devasyarajguru**
# Title and Issue number 

Title: Improve UI to Birthday Website
Issue Number:  #1013 

Close #1013 

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have created a helpful and easy to understand `README.md`

birthday

### CODE REVIEW

**Comment by devasyarajguru** [comment]
Hey @pranjay-poddar Please review my PR and if it goes well please merge it. Also please add level-2 tag. Thank You!

**Comment by devasyarajguru** [comment]
@Saurabh-kumar09 Please review my PR and if it goes well please merge it. Also please add level-2 tag. Thank You!

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
@devasyarajguru please resolves conflicts to merge this PR

**Comment by devasyarajguru** [comment]
@MohitGupta121 As per requested changes are made , commit message is - "Merge branch main into devasya23" mentioned above in the conversation. Please check it and merge my PR

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
@devasyarajguru  please don't upload directly too much large size files fetch or use them from cloud url such as 1st you upload your video in google drive or directly in Github, after that you get 1 link use that link in your project, not use directly images, videos files.
This increases the whole Dev-Greek project size.
Thanks!

File: Frontend-Projects/birthday website/balloons.jpg
Comment (MohitGupta121) [inline Frontend-Projects/birthday website/balloons.jpg:1]: @devasyarajguru I am saying to use these image files from the cloud or use its link. Don't upload the images in the PR. Fetch them from google drive or any other cloud image link.

File: Frontend-Projects/birthday website/balloons1.jpg
Comment (MohitGupta121) [inline Frontend-Projects/birthday website/balloons1.jpg:1]: same for this images.

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

**Comment by devasyarajguru** [comment]
Hey @MohitGupta121 , I have resolved my conflicts in my VS Code. After that I did commit and try to push the local changes to the repo, but it did'nt work out , I am getting errors which I tried to search solutions for it but did'nt solve any. Please help me with it, I am posting screenshot of errors below. 

### **`No. 1 `**- 

### **`No.2`**


### **`No.3`**



### **`No.4`**


### **`No.5`**


### **`No.6`**

**Comment by MohitGupta121** [comment]
@devasyarajguru you also have to update your branch with main upstream branch then this will resolve

**Pull request closed without merging**